### DISCUSSION TITLE
Publish a companion app on the AppCenter for the EDW

### DISCUSSION THREAD

**Discussion by JoseExposito** [elementary/edw#35] [Ideas]
Create an app with useful information about the EDW, talk schedule and links.

This application could be used to illustrate many topics covered in other talks like, UI and icon design, translation process, Flatpak manifest creation, publishing on the AppCenter, etc

Speaker: None or TBD

**Comment by jalcine** [comment]
Slightly unrelated, I know elementary doesn't do something like what I'm about to propose _now_ but I wouldn't be opposed to them having a _general_ notification system / application like this. I know that Apple does it and it's really the only way I find out about new _changes_ to the platform. Something like this could push big release notes (or even announce this event and give tracking info)
